migrations/20191121013724-initial_table_creation.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    /*
+      Add altering commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      return queryInterface.createTable('users', { id: Sequelize.INTEGER });
+    */
+    return queryInterface.createTable('teams', {
+      id: {
+        type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true,
+      },
+      location: { type: Sequelize.STRING },
+      mascot: { type: Sequelize.STRING },
+      abbreviation: { type: Sequelize.STRING },
+      conference: { type: Sequelize.STRING },
+      division: { type: Sequelize.STRING },
+      createdAt: { type: Sequelize.DATE, defaultValue: Sequelize.literal('CURRENT_TIMESTAMP'), },
+      updatedAt: { type: Sequelize.DATE, defaultValue: Sequelize.literal('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'), },
+      deletedAt: { type: Sequelize.DATE },
+    })
+    //return queryInterface.createTable('teams')
+  },
+
+  down: (queryInterface, Sequelize) => {
+    /*
+      Add reverting commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      return queryInterface.dropTable('users');
+    */
+    queryInterface.dropTable('teams', {
+      id: {
+        type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true,
+      },
+      location: { type: Sequelize.STRING },
+      mascot: { type: Sequelize.STRING },
+      abbreviation: { type: Sequelize.STRING },
+      conference: { type: Sequelize.STRING },
+      division: { type: Sequelize.STRING },
+      createdAt: { type: Sequelize.DATE, defaultValue: Sequelize.literal('CURRENT_TIMESTAMP'), },
+      updatedAt: { type: Sequelize.DATE, defaultValue: Sequelize.literal('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'), },
+      deletedAt: { type: Sequelize.DATE },
+    })
```

Comment:
This isn't needed, you only need to drop the table once and you don't need to provide the structure of the table when you do. The reason this doesn't throw an error is because you aren't `await`ing this first drop. If you had been the second drop would throw an error since the table would have been dropped by the first.